api/proto/node/node.proto

```diff
@@ -105,15 +105,33 @@ message Bundle {
 	repeated bytes chunks = 1;
 }
 
+message G1Commitment {
```

Comment:
This G1Commitment could be shared in disperser.proto for the KZG commitment, so it could be a unified type